db/models.py

```diff
@@ -1 +1,46 @@
 from django.db import models
+
+
+class Race(models.Model):
+    race_to_choose = [
+        ["Elf", "Elf"],
+        ["Dwarf", "Dwarf"],
+        ["Human", "Human"],
+        ["Ork", "Ork"]
+    ]
```

Comment:
The `choices` attribute in the `Race` model should use tuples instead of lists. Change `race_to_choose` to use tuples like this: `('Elf', 'Elf')`.